roles/docker/tasks/systemcontainer_docker.yml

```diff
@@ -13,10 +13,12 @@
       in your inventory file.
 
 - name: Ensure container-selinux is installed
+  when: not openshift.common.is_atomic | bool
   package:
     name: container-selinux
     state: present
-  when: not openshift.common.is_atomic | bool
+  register: l_result
+  until: l_result.rc is defined and l_result.rc == 0
```

Comment:
"retries" is actually optional and defaults to 3. I like having it there so it's obvious what's happening but it is an extra line for each of these...